client/yarn.lock

```diff
@@ -1361,7 +1361,7 @@ color-string@^1.5.2:
     color-name "^1.0.0"
     simple-swizzle "^0.2.2"
 
-color@^2.0.1:
+color@^2.0.0, color@^2.0.1:
```

Comment:
any idea what the reasoning behind this is?

Reply:
I don't remember doing this - I will revert.